app/containers/UIKit/MultiSelect/Chips.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+	View, Text, FlatList, Image
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Touchable from 'react-native-platform-touchable';
+
+import { themes } from '../../../constants/colors';
+import { textParser } from '../utils';
+import { CustomIcon } from '../../../lib/Icons';
+
+import styles from './styles';
+
+const keyExtractor = item => item.value.toString();
+
+const Chip = ({ item, onSelect, theme }) => (
+	<Touchable
+		key={item.value}
+		onPress={() => onSelect(item)}
+		style={[styles.chip, { backgroundColor: themes[theme].auxiliaryBackground }]}
+		background={Touchable.Ripple(themes[theme].bannerBackground)}
+	>
+		<>
+			{item.imageUrl ? <Image style={styles.chipImage} source={{ uri: item.imageUrl }} /> : null}
+			<Text style={[styles.chipText, { color: themes[theme].titleText }]}>{textParser([item.text])}</Text>
+			<CustomIcon name='cross' size={16} color={themes[theme].auxiliaryText} />
+		</>
+	</Touchable>
+);
+Chip.propTypes = {
+	item: PropTypes.object,
+	onSelect: PropTypes.func,
+	theme: PropTypes.string
+};
+
+const Chips = ({ items, onSelect, theme }) => (
+	<View>
+		<FlatList
+			data={items}
+			keyExtractor={keyExtractor}
```

Comment:
Maybe these chips should wrap in case of several chips selected.
Also, there's no way to one more chip in this case and the scroll is weird because on the touch.
![Feb-04-2020 13-28-51](https://user-images.githubusercontent.com/804994/73765143-e13d4200-4752-11ea-9793-19f35a9d3076.gif)